Tests/SourceKittenFrameworkTests/DocInfoTests.swift

```diff
@@ -22,9 +22,9 @@ class DocInfoTests: XCTestCase {
     }
 
     func testModuleInfoRequest() throws {
-#if swift(>=4.1.50)
+#if swift(>=4.2)
```

Comment:
Why this change? Same compiler version, no?

Reply:
By this PR, SourceKitten will be built with `-swift-version 4.2` swift compiler option.
Under that condition, it will never be `4.1.50`, so I thought that this should be changed.
🤔 
Or should we switch to [compiler version directive](https://github.com/apple/swift-evolution/blob/master/proposals/0212-compiler-version-directive.md) on occasion?

Reply:
In some cases maybe we should but that’s unnecessary right now.